scripts/nolints.txt

```diff
@@ -1206,8 +1206,8 @@ apply_nolint add_con.to_setoid doc_blame
 apply_nolint con.to_setoid doc_blame
 
 -- group_theory/coset.lean
-apply_nolint is_add_subgroup.group_equiv_quotient_times_subgroup doc_blame
-apply_nolint is_add_subgroup.left_coset_equiv_subgroup doc_blame
+apply_nolint is_add_subgroup.add_group_equiv_quotient_times_add_subgroup doc_blame
+apply_nolint is_add_subgroup.left_coset_equiv_add_subgroup doc_blame
```

Comment:
Contrary to normal practice, I'm adjusting `nolints.txt` directly here. These are automatically generated definitions whose automatic names are changed by this improvement to `to_additive`. Moreover, they are deprecated and planned for obsolesence, so I don't want to write doc-strings. :-)